### ISSUE TITLE
Phalcon\Http\Response getStatusCode() returns full string instead of int code

### ISSUE THREAD

**Issue by stamster** [phalcon/cphalcon#12895]
### Expected and Actual Behavior

> 
>        $http_status = 422;
>        $this->response->setStatusCode($http_status);
>       var_dump($this->response->getStatusCode());

Expected:  `int(422) or string(3) "422" `

Actual result is:
` string(24) "422 Unprocessable Entity"`

> Provide minimal script to reproduce the issue

```php
$res = new Phalcon\Http\Response();
$res->setStatusCode(422);
var_dump($res->getStatusCode());
```
### Details

* Phalcon version: 3.1.2
* PHP Version: 7.0.18
* Operating System: GNU/Linux, Ubuntu 16.04 LTS
* Installation type: Compiling from source
* Zephir version (if any): 0.9.7-1fae5e50ac
* Server: Nginx
* Other related info (Database, table schema): n/a

Exactly same code works w/o this issue on PHP 5.6.30 and same Phalcon version / setup.

**Comment by stamster** [comment]
Thanks @sergeyklay 

It amazes me how this is different between PHP versions, and not Phalcon versions.

**Comment by Jurigag** [comment]
`headers->set("Status", code . " " . message);` fomr setStatusCode

`return this->getHeaders()->get("Status");` from getStatusCode, from code it should return really `422 Unprocessable Entity`

**Comment by stamster** [comment]
Well now I'm not sure whenever the issue was PHP 5 or my Database default values. More precisely, if database had column defined as int  `string(24) "422 Unprocessable Entity"` will get trimmed out by default and database will only have int 422 saved in that row.

Nevertheless, this is definitely a bug - now confirmed exact same behavior on both PHP 5.6.30 and PHP 7.0.x. As we can see in zephir sources, surely this method will return full string, which is not desirable.

**Comment by sergeyklay** [comment]
Fixed in the `3.2.x branch`. Thank you for contributing.
